src/Accelerators/NNPA/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(DLC_SRC_ROOT "${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Do all the `DLC_` properties need to be set? If not, they are not really needed.

Reply:
I believe they do need to be set, but I will check